AutomatorServer/android/app/src/androidTest/java/pl/leancode/automatorserver/PatrolAutomator.kt

```diff
@@ -152,20 +147,23 @@ class PatrolAutomator private constructor() {
         return uiObjects2.map { fromUiObject2(it) }
     }
 
-    fun tap(selector: UiSelector) {
-        Logger.d("tap() selector $selector")
+    fun tap(uiSelector: UiSelector, bySelector: BySelector) {
+        Logger.d("tap() selector $uiSelector")
```

Comment:
Update the log messages in this file to include the `bySelector`s too

Reply:
good idea